src/group/mls_group/mod.rs

```diff
@@ -288,11 +288,11 @@ impl MlsGroup {
     }
 
     /// Get the ciphersuite implementation used in this group.
-    pub(crate) fn ciphersuite(&self) -> &Ciphersuite {
+    pub fn ciphersuite(&self) -> &Ciphersuite {
         self.ciphersuite
     }
 
-    pub(crate) fn context(&self) -> &GroupContext {
```

Comment:
Getting the context is (as far as I know) the only way to obtain the `GroupId`. I think the outside application should have some way identifier for the group and it might as well be the `GroupId`. This is probably going to be needed in other cases as well, for example if a client wants to let the server know what groups they are a member of.

Reply:
Ideally there would be getters for the fields of the group context so that they don't have to be public.